keras/layers/rnn/bidirectional_test.py

```diff
@@ -1020,6 +1020,20 @@ def test_full_input_spec(self):
         self.assertAllClose(output1, output3)
         self.assertNotAllClose(output1, output2)
 
+    def test_trainable_parameter_argument(self):
+        inp = keras.layers.Input([None, 3])
+        rnn = keras.layers.SimpleRNN(units=3)
+        bid = keras.layers.Bidirectional(rnn)
+        model = keras.Model(inp, bid(inp))
+
+        clone_trainable = keras.models.clone_model(model)
+        assert clone_trainable.get_config() == model.get_config()
+
+        bid.trainable = False
+
+        clone_untrainable = keras.models.clone_model(model)
+        assert clone_untrainable.get_config() == model.get_config()
+
```

Comment:
Can you also add a test to create a `keras.layers.Bidirectional` with a trainable = False in the __init__ arg?